src/runtime/hack/update-generated-hypervisor-proto.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -o errexit -o pipefail -o nounset
+
+cd "$(dirname "${BASH_SOURCE[0]}")/.."
+
+protoc --gogottrpc_out=protocols/hypervisor \
```

Comment:
There is this effort to migrate out of gogo/protobuf (https://github.com/kata-containers/kata-containers/issues/7420) performed by @beraldoleal . I'd like to have Beraldo to look at this change to ensure we are not introducing more gogo protobuf that should be removed later.

Reply:
I think Yohei followed the model of https://github.com/kata-containers/kata-containers/tree/main/src/runtime/protocols/cache, but if we want to move it somewhere else that's fine with me

Reply:
Hi @wainersm, thank you for ping me here.

While it is not adding any gogo specifics to the .proto file and its compiled artifacts, it is adding a new script that uses `--gogottrpc`. Currently, our other scripts are doing the same, but there is indeed an effort to drop it. Instead of `--gogottrpc` we should start using `--go-ttrpc_out`.

I'm not sure tho, at this point, if we can live with both at same time. So I'm fine with this as it is, and I can drop it when doing the removal in other places so we can drop it all at once.

Reply:
Now, about the location, I miss the `option go_package`, to understand where those should be placed after compiled.

All .proto files under `src/libs/protocols/protos` has that. And will need that soon on later.

Reply:
Thank you @stevenhorsman for adding the `go_package` option. Yes, if cache is there, I don't mind having another one. But we should start a discussion to make sure the current layout makes sense.

Reply:
Hi @stevenhorsman since #8061 was merged already, I guess we have to migrate this away from gogo here now. Let me know if you need some help with it.

Reply:
Hey Beraldo - thanks for reminding me about this. I've tried to do the migration, but if you have a chance to review what I've done and see if it works out then that would be appreciated!

Reply:
Closing this conversation as Beraldo review it already.